index.php

```diff
@@ -42,8 +42,7 @@
                 <div id="amounts">
                     <?php
                         $paidString = 'has_paid_'.$config['release_title'].'_'.$config['release_version'];
-                        $disallowed = [' ', '.'];
-                        $encoded = urlencode(str_replace($disallowed, '_', $paidString));
+                        $encoded = urlencode($disallowed, '_', $paidString);
```

Comment:
You are referencing `$disallowed` but you removed the line defining it. Sure you didn't mean `urlencode($paidString);`?